superset-frontend/src/views/CRUD/data/query/QueryList.tsx

```diff
@@ -0,0 +1,356 @@
+    cursor: pointer;
+  }
+`;
+
+const StyledPopoverItem = styled.div`
+  color: ${({ theme }) => theme.colors.grayscale.dark2};
+`;
+
+function QueryList({ addDangerToast, addSuccessToast }: QueryListProps) {
+  const {
+    state: { loading, resourceCount: queryCount, resourceCollection: queries },
+    fetchData,
+  } = useListViewResource<QueryObject>(
+    'query',
+    t('Query History'),
+    addDangerToast,
+    false,
+  );
+
+  const menuData: SubMenuProps = {
+    activeChild: 'Query History',
+    ...commonMenuData,
+  };
+
+  const initialSort = [{ id: 'changed_on', desc: true }];
+  const columns = useMemo(
+    () => [
+      {
+        Cell: ({
+          row: {
+            original: { status },
+          },
+        }: any) => {
+          const statusConfig = {
+            name: '',
+            label: '',
+            color: '',
+          };
+          if (status === 'success') {
+            statusConfig.name = 'check';
+            statusConfig.label = t('Success');
+            statusConfig.color = '#59C189';
+          }
+          if (status === 'failed' || status === 'stopped') {
+            statusConfig.name = 'x-small';
+            statusConfig.label = t('Failed');
+            statusConfig.color = '#E04355';
+          }
+          if (status === 'running') {
+            statusConfig.name = 'running';
+            statusConfig.label = t('Running');
+            statusConfig.color = '#20A7C9';
+          }
+          if (status === 'timed_out') {
+            statusConfig.name = 'offline';
+            statusConfig.label = t('Offline');
+            statusConfig.color = '#B2B2B2';
+          }
+          if (status === 'scheduled' || status === 'pending') {
+            statusConfig.name = 'queued';
+            statusConfig.label = t('Scheduled');
+            statusConfig.color = '#666666';
+          }
+          return (
+            <Tooltip title={statusConfig.label} placement="bottom">
+              <Icon
+                name={statusConfig.name as IconName}
+                color={statusConfig.color}
+              />
+            </Tooltip>
+          );
+        },
+        accessor: 'status',
+        size: 'xs',
+        disableSortBy: true,
+      },
+      {
+        accessor: 'start_time',
+        Header: t('Time'),
+        size: 'xl',
+        Cell: ({
+          row: {
+            original: { start_time, end_time },
+          },
+        }: any) => {
+          const startDate = new Date(start_time);
+          const startUtc = new Date(
+            Date.UTC(
+              startDate.getFullYear(),
+              startDate.getMonth(),
+              startDate.getDate(),
+              startDate.getHours(),
+              startDate.getMinutes(),
+              startDate.getSeconds(),
+              startDate.getMilliseconds(),
+            ),
+          );
+          const startMoment = moment(startUtc);
+          const formatedStartTime = startMoment.format('YYYY-MM-DD HH:mm:ssZ');
```

Comment:
pull the format string into a constant to be used anywhere we want to format these times?